Documentation/report.md

```diff
@@ -297,3 +297,29 @@ A report can have the following states:
 
 
 [rfc3339]: https://tools.ietf.org/html/rfc3339#section-5.8
+
+## Roll-up Reports
+
+Report data is stored in the database much like metrics themselves, and can thus be used in aggregated or roll-up reports. A simple use case for a roll-up report is to spread processing load across a period of time: instead of running a monthly report all at once, it can instead be a simple sum of daily reports.
+
+A custom roll-up report requires a custom generation query. The ReportGenerationQuery processor provides a macro that can get the necessary table name from a report name:
```

Comment:
Can you perhaps make the `from a report name` text link to rollup-reports.md at the section where this query is introduced? It's not super obvious where `namespace-cpu-usage-aggregated-query.yaml` comes from since the link comes after these snippets. 

I think this would be good:

```
The ReportGenerationQuery processor provides a macro that can get the necessary table name [from a report name](rollup-reports.md#2-create-the-aggregation-query):
```

The ReportGenerationQuery processor provides a macro that can get the necessary table name [from a report name](rollup-reports.md#2-create-the-aggregation-query):